docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>this example is rather involved. Can we pare it down further? For example: we don't need to post select; then <code>z_density_From_measurements</code>  is just a call to <code>np.average</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>All the tests we did were with post selection. If we want to strip down the post selection I'd suggest to tests this on hardware first.</p>

Reply:
I simplified the "helper function" section by combining post-selection and averaging. @mpharrigan do you think it's more reasonable now?